deploy/openshift/quay-py3-app.yaml

```diff
@@ -393,6 +395,9 @@ parameters:
   - name: QUAY_APP_READINESS_PROBE_TIMEOUT_SECONDS
     value: "10"
     displayName: quay app readiness probe timeout
+  - name: QUAY_DISABLE_MIN_THREE_REPLICAS_CHECK
+    value: ""
```

Comment:
one thing I'm not sure is if having an empty value keeps the check enabled. We need to validate this in stage before we deploy in prod.

Reply:
or we can even evaluate this in our dev cluster

Reply:
@syed sounds good to me, fwiw [the doc](https://github.com/stackrox/kube-linter/blob/main/docs/configuring-kubelinter.md#ignoring-violations-for-specific-cases) I was looking at suggests that an empty value will still disable the check

Reply:
ok that might be an issue here. If that's the case, can you also make a param of the annotation key as well? `ignore-check.kube-linter.io/minimum-replicas` will become a param with default empty value

Reply:
done 🙂